src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerPushImage.groovy

```diff
@@ -56,7 +57,7 @@ class DockerPushImage extends AbstractDockerRemoteApiTask implements RegistryCre
 
         for (String image : images.get()) {
             AuthConfig authConfig = getRegistryAuthLocator().lookupAuthConfig(image, registryCredentials)
-            logger.quiet "Pushing image '${image}' to ${authConfig.registryAddress}."
+            logger.quiet "Pushing image '${image}' to ${RegistryAuthLocator.getRegistry(image)}."
```

Comment:
Should use the instance of `getRegistryAuthLocator()`.

Reply:
The same logic should be used in `DockerPullImage`.

Reply:
Is it fine to call `getRegistryAuthLocator()`, or should I pull that out to a variable locally?

Reply:
I'd assign it to a variable for reuse within the same class.